### PR TITLE
About us page

### PR DESCRIPTION
**Pull request by Lily-s**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,9 +1,63 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Saira:ital,wght@0,100..900;1,100..900&display=swap"
+      rel="stylesheet"
+    />
+    <link href="css/headerFooter.css" rel="stylesheet" />
+    <link href="css/aboutUs.css" rel="stylesheet" />
+
     <meta charset="UTF-8" />
```
Comment (BeetrootAcademyAccount3) [inline index.html:13]: Meta tags are before style links in the hierarchy

File: index.html [hunk 2]
```diff
@@ -1,9 +1,63 @@
...
+      <div id="paw"></div>
+      <a href="#" id="aboutUs">About us</a>
+      <a href="#" id="menu">Menu</a>
+      <a href="#" id="careers">Careers</a>
+      <a href="#" id="buttonOrder">ORDER NOW</a>
+    </header>
+    <main>
+      <div class="container">
+        <div class="about-founder">
+          <img src="../assets/svg/original.svg" alt="Bear" class="bear-image" />
```
Comment (BeetrootAcademyAccount3) [inline index.html:28]: Wrong reference for images, it should start  with "./"

Capture

File: index.html [hunk 3]
```diff
@@ -1,9 +1,63 @@
...
+          <div class="founder-text">
```
Comment (BeetrootAcademyAccount3) [inline index.html:29]: These divs can be sections in order for the page to be better semantically

File: css/headerFooter.scss [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+.calibriFont {
+  font-family: Calibri, Candara, Segoe, "Segoe UI", Optima, Arial, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 20px;
+}
+
+.header {
```
Comment (BeetrootAcademyAccount3) [inline css/headerFooter.scss:22]: Currently the content does not occupy the whole page. Check how can we achieve that and to set the footer to the bottom. Maybe position:fixed will be a better solution :)

Capture

**Review by BeetrootAcademyAccount3 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Lily-s) [reply index.html:13]: fixed

File: index.html (hunk 2 above)
Comment (Lily-s) [reply index.html:28]: fixed

File: index.html (hunk 3 above)
Comment (Lily-s) [reply index.html:29]: fixed

File: css/headerFooter.scss (hunk 4 above)
Comment (Lily-s) [reply css/headerFooter.scss:22]: I have a problem here. If I make the position fixed, main content is not stretched up to the bottom of the page. How to deal with it?

Comment (Lily-s) [reply css/headerFooter.scss:22]: image

Comment (Lily-s) [reply css/headerFooter.scss:22]: hm...you know, I think I've got it with this property
min-height: 100vh;

File: index.html [hunk 5]
```diff
@@ -1,6 +1,8 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (BlackFlame-hub) [inline index.html:5]: Changes have been made

**Review by BlackFlame-hub — DISMISSED** [review]
The recommended changes have been made, all seems fine.

**Review by BlackFlame-hub — APPROVED** [review]

**Review by BeetrootAcademyAccount3 — APPROVED** [review]

**Comment by BeetrootAcademyAccount3** [comment]
Since this is now merged to main, update it locally on your machines

**Pull request merged**